### PR TITLE
New method to get scripts that doesn't involve a load of duplication

### PR DESCRIPTION
**Pull request by JackMc**
As discussed in comments of #225 

This is just an idea of mine, feel free to fork/CR

### CODE REVIEW

**Comment by dborzov** [comment]
:+1:

File: datacats/docker.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 from __future__ import absolute_import
 
+from datacats import scripts
```
Comment (wardi) [inline datacats/docker.py]: I think `from datacats.scripts import get_script_path` is warranted here

File: datacats/docker.py [hunk 2]
```diff
@@ -363,7 +363,9 @@ def collect_logs(name):
 
 def check_connectivity():
     c = run_container(None, 'datacats/web', '/project/check_connectivity.sh',
```
Comment (wardi) [inline datacats/docker.py]: use user_run_script please (i know, not from this change...)

**Comment by wardi** [comment]
otherwise looks good!

File: datacats/docker.py (hunk 2 above)
Comment (JackMc) [inline datacats/docker.py]: this is in docker.py, using user_run_script would cause a circular dep, no?

**Comment by JackMc** [comment]
Closing to reopen on branch

**Pull request closed without merging**